pkg/inject/template-values.go

```diff
@@ -0,0 +1,91 @@
+package inject
+
+type (
+	// InjectValues contains the top-level elements in the Helm charts
+	InjectValues struct {
+		Namespace        string
+		ClusterDomain    string
+		HighAvailability bool
+		Identity         *Identity
+
+		Proxy     *Proxy
+		ProxyInit *ProxyInit
+	}
+
+	// Proxy contains the fields to set the proxy sidecar container
+	Proxy struct {
+		Capabilities          *Capabilities
+		Component             string
+		DisableIdentity       bool
+		DisableTap            bool
+		EnableExternalProfile bool
+		Image                 *Image
+		LogLevel              string
+		SAMountPath           *SAMountPath
+		Ports                 *Ports
+		Resources             *Resources
+		UID                   int64
+	}
+
+	// ProxyInit contains the fields to set the proxy-init container
+	ProxyInit struct {
+		Capabilities        *Capabilities
+		IgnoreInboundPorts  string
```

Comment:
can should this be an array?

Reply:
yeah we might. WDYT @ihcsim ?

Reply:
The value of this variable eventually gets mapped into the `args` of the proxy-init container. From a rendering perspective, I think it should remain as string. Otherwise, wont' we end up with something like:

```
  initContainers:
  - args:
    ...
    - --inbound-ports-to-ignore
    - [4190,4191]
    - --outbound-ports-to-ignore
    - ["443"] 
```